src/main/java/org/apache/sling/xss/impl/HtmlToHtmlContentContext.java

```diff
@@ -62,21 +58,17 @@ public boolean check(final PolicyHandler policyHandler, final String str) {
      * @see XSSFilterRule#filter(PolicyHandler, java.lang.String)
      */
     @Override
-    public String filter(final PolicyHandler policyHandler, final String str) {
-        if (StringUtils.isNotEmpty(str)) {
+    public String filter(final PolicyHandler policyHandler, final String malicousString) {
```

Comment:
Typo: I think you meant malicious? I think it's better to use something like `unsafe` or `unsanitized` though, we are not sure it's malicious at this point.

Reply:
Resolved: 7d9e82449f514b2fce6ba4c3aa4ec0df70daf8bb